cpp/src/arrow/compute/kernels/scalar_temporal_test.cc

```diff
@@ -566,6 +566,14 @@ TEST_F(ScalarTemporalTest, Strftime) {
                    utf8(), string_microseconds, &options);
   CheckScalarUnary("strftime", timestamp(TimeUnit::NANO, "US/Hawaii"), nanoseconds,
                    utf8(), string_nanoseconds, &options);
+
+  auto options2 = StrftimeOptions("%Y-%m-%d");
```

Comment:
Can you also add a test with formatting hours/minutes/seconds? (I suppose those will just work, with all 0?)

Reply:
Will add. I suspect so.

Reply:
Done.